src/ctia/entity/event/store.clj

```diff
@@ -14,6 +14,7 @@
     (crud/handle-create state new-events))
   (list-events [this filter-map ident params]
     (crud/handle-list state filter-map ident params))
+  (close [_] (close-cm! state))
```

Comment:
Perhaps, at some point we could discuss renaming `ctia.stores.es.store/close-cm!`. Arguably, `cm` maybe not very obvious abbreviation,  without jumping to where the fn is implemented, it's not clear. But, anyone who reads the code has to do that only once, so, really not that big of a deal.

Reply:
I agree. I always prefer easier code to read. I replaced it by `close-connections` (indeed it closes a connection manager).